ckanext/xloader/config_declaration.yaml

```diff
@@ -13,9 +13,12 @@ groups:
       - key: ckanext.xloader.user
         example: xloader_system
         description: |
-            The username of a sysadmin that will be executing the actions of the
-            xloader service. This user should also have an API token to download
-            private resource files.
+            The username of that will be executing the actions of the xloader service.
+            This user should have higher permissions.
+      - key: ckanext.xloader.api_token
+        example: eyJ0eXAiOiJKV1QiLCJh.eyJqdGkiOiJ0M2VNUFlQWFg0VU.8QgV8em4RA
+        description: |
+            Uses a specific API token for the downloading private resources.
```

Comment:
Why do we still need a token for downloading private resources when we can run actions locally?  Is there no action to return a real file url that we can pass `ignore_auth` to?

Reply:
Yeah, we can rewrite how Xloader retrieves files too

Reply:
@wardi okay this works a lot better and makes more sense. It uses the `get_resource_uploader` method now. For cloud stroage, it relies on this PR though: https://github.com/open-data/ckanext-cloudstorage/pull/9

It also looks like the core `get_path` method does not need any context, it will just return the actual file storage path no matter what. So this would work with that too